wgsl/index.bs

```diff
@@ -3978,7 +3978,7 @@ effective-value-type.
     converted=] to the [=effective-value-type=].
 3. If an initializer is not specified, a value must be provided at [=pipeline
     creation|pipeline-creation time=].
-4. [=Override-declarations=] are part of the pipeline interface, but are not
+4. [=Override-declarations=] are part of the shader interface, but are not
```

Comment:
Would this better be shader stage interface or more correct to leave as shader interface @alan-baker ?

Reply:
Good question.  They are set on the pipeline, so I prefer it to say "shader interface" here.  But that's a conceptual judgement.

Reply:
@dneto0 thanks